windows/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_library(hidapi_winapi
+    ${HIDAPI_PUBLIC_HEADERS}
+    hid.c
+)
+target_link_libraries(hidapi_winapi
+    PUBLIC hidapi_include
+    PRIVATE Setupapi
```

Comment:
mingw on Linux is case sensitive.
```suggestion
    PRIVATE setupapi
```

Reply:
and this is the caseness those use? The "correct" one (as per WindowsSDK) is even different:
![image](https://user-images.githubusercontent.com/5939659/114893523-74254200-9e16-11eb-8581-c90e73d2c740.png)

Reply:
This line indicates `libsetupapi.a` is generated:
https://github.com/mirror/mingw-w64/blob/c6e13e0c105eab7797c2373819b49fff6b05566c/mingw-w64-crt/lib64/Makefile.am#L539

Reply:
as much as I personally don't like the idea of tuning a "Windows" backends for non-windows build environments, I guess an ideology itself is not a very good reason for keeping Linux/mingw off-the-board

Reply:
This change will benefit all cross building.
The problem would appear as well when building using msys2 on Windows.